src/pages/map.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+  Tab,
+  Tabs,
+  TabList,
+  TabPanel,
+  TabPanels,
+  Heading,
+  Stack,
+  Box,
+  Text,
+  Spinner,
+} from '@chakra-ui/react';
+import React, { useState } from 'react';
+import { trpcNext } from "../trpc";
+import { Landmark } from 'shared/Map';
+
+export default function Page() {
+  const [tabIndex, setTabIndex] = useState(0);
```

Comment:
Use TabsWithUrlParam.tsx and not save tab state locally. Take a look at how the other call sites of TabsWithUrlParam uses it.